tools/vsimporter/xib2nib/UIButton.cpp

```diff
@@ -253,22 +245,15 @@ void UIButton::InitFromStory(XIBObject* obj) {
     if (type) {
         if (strcmp(type, "roundedRect") == 0) {
             getAttrAndHandle("buttonType");
+             // TODO: Investigate why this value deviates from UIButtonType
```

Comment:
We'd have to crack open the NIB from iOS and inspect its values for various UIButton types to see if they align with UIButtonType*. Right now, I don't know if this value is correct - might be a legacy value or just a hack.

---
In reply to: [102358917](https://github.com/Microsoft/WinObjC/pull/2018#discussion_r102358917) [](ancestors = 102358917,102358850)